src/Commands/Modules/Configuration/ConfigurationCommand.cs

```diff
@@ -1,32 +1,97 @@
-﻿using System.Linq;
+﻿using System;
+using System.Linq;
 using System.Threading.Tasks;
 
 using Discord;
 using Discord.Commands;
+using Discord.WebSocket;
 
 using BrackeysBot.Services;
 
 namespace BrackeysBot.Commands
 {
     public partial class ConfigurationModule : BrackeysBotModule
     {
-        public ConfigurationService Config { get; set; }
+        private IUserMessage _lastConfigMessage;
```

Comment:
So what happens if 2 people use []config in different channels?

Reply:
It still works individually for each embed as it subscribes to different versions of the event.